### DISCUSSION TITLE
Is the Slack channel still actively used?

### DISCUSSION THREAD

**Discussion by n-stein** [moneymanagerex/moneymanagerex#5750] [Q&A]
Basically the title. The Slack link on moneymanagerex.org is broken, it takes you to http://slack.moneymanagerex.org. The link from the app itself takes you to https://moneymanagerex.slack.com which appears to work but asks for a rogers.com email to sign up. If it is actively used, how does one gain access?

**Comment by tactilis** [comment]
I just logged in to Slack (first time in several months). It took ages to load but eventually:

![image](https://user-images.githubusercontent.com/4060092/224532684-14af5962-e602-4291-a4e5-e921a892f551.png)

![image](https://user-images.githubusercontent.com/4060092/224532724-92615a09-7d6f-46cc-8386-a0a8a315fcbd.png)

**Comment by tactilis** [comment]
There's very little activity though, which is why I stopped visiting.

People seem to use the MMEX forum and github issues/discussions much more.

**Comment by tactilis** [comment]
> The Slack link on moneymanagerex.org is broken, it takes you to http://slack.moneymanagerex.org/.

Confirmed.

> The link from the app itself takes you to https://moneymanagerex.slack.com/ which appears to work

It does.


> but asks for a rogers.com email to sign up.

I don't experience this.

> If it is actively used, how does one gain access?

Sign up for a Slack account. Go to https://moneymanagerex.slack.com/

**Comment by n-stein** [comment]
> Sign up for a Slack account. Go to https://moneymanagerex.slack.com/

This is the part that doesn't work. I am given the message 

> Please use an @rogers.com email address to join this workspace without an invitation.

**Comment by tactilis** [comment]
Does your browser have any cookies or username/passwords saved for slack.com?

What happens if you go to slack.com in a private/incognito window? Or using a different browser?

**Reply by n-stein** [reply]
> Does your browser have any cookies or username/passwords saved for slack.com?
> 
> What happens if you go to slack.com in a private/incognito window? Or using a different browser?

No, in fact I've never used slack.com so that's why I was wondering if I'm doing something wrong. I'm going to https://moneymanagerex.slack.com and clicking on the "Create an account" link
![image](https://user-images.githubusercontent.com/16599929/228365395-461ab85f-4d18-417d-973f-5756e1588c7d.png)

And every browser -- Chrome, Edge, Firefox -- and even Slack's mobile app all give me the same message:
![image](https://user-images.githubusercontent.com/16599929/228364984-7f920692-811e-49c7-bd38-a4c604e2bb92.png)

If it isn't used actively I'll just cease my attempt to join and stick with GitHub.

**Reply by renato-mmex** [reply]
And if you download the app here:   https://slack.com/intl/de-ch/downloads/windows  or in the Microsoft Store

**Reply by nmsleal** [reply]
I have the same "@rogers" limitation (Chrome, Ubuntu). But it seems to be not worthy, so I'll keep using GitHub. I guess Slack is losing momentum generally...

**Reply by renato-mmex** [reply]
Send me your email address to mmex-reg@bluewin.ch and I will send you an invitation.

**Comment by renato-mmex** [comment]
Slack is rarely used by MMEX users.
For me it doesn't do much, the only information I appreciate in Slack is the notification of completed "pull requests"
